src/ducks/apps/components/ApplicationDetails.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import Icon from 'cozy-ui/react/Icon'
+
+import defaultAppIcon from '../../../assets/icons/icon-cube.svg'
+
+const ApplicationDetails = ({t, app: { description, icon, installed, name, related, slug }, openApp, parent}) => (
+  <div className='sto-app'>
+    <div className='sto-app-icon'>
+      {
+        icon
+        ? <img src={icon} alt={`${slug}-icon`} width='170' />
+        : <svg className='sto-small-app-item-icon--default blurry' width='170' style='padding: 0 8px'>
```

Comment:
Could you put the style in the css to avoid inline style?

Reply:
Done !